### ISSUE TITLE
Ask yt-dlp to return only a single video from standard URLs

### ISSUE THREAD

**Issue by kietheros** [yt-dlp/yt-dlp#2773]
### Checklist

- [X] I'm asking a question and **not** reporting a bug/feature request
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions including closed ones

### Question

I want to download only one video from standard youtube URLs (only relate to one video)
https://www.youtube.com/watch?v=-wtIMTCHWuI
https://youtu.be/-wtIMTCHWuI
https://www.youtube.com/v/-wtIMTCHWuI?version=3&autohide=1

**Ignore these links: channel, playlists,...**
+ URL 1: https://www.youtube.com/user/IngridMichaelsonVEVO#p/a/u/1/QdK8U-VIH_o
+ URL 2: https://www.youtube.com/results?search_query=shake+it+off

I'm testing with this command: 
`yt-dlp --dump-json --no-playlist --youtube-skip-dash-manifest [URL] > test.json`
+ URL 1: the result is an invalid json file. It contains **2 objects**.
+ URL 2: the command runs too slowly because it parses many video items.

In short, I want to ask yt-dlp to do:
+ Only work with standard youtube URLs, and ignore URLs which contain playlists 
+ **OR**, always return only one video from above URLs.

**Comment by pukkandan** [comment]
> - [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)

- See "video selection" section of readme
